src/current/_includes/releases/v23.2/v23.2.0-beta.3.md

```diff
@@ -0,0 +1,43 @@
+## v23.2.0-beta.3
+
+Release Date: December 13, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-2-0-beta-3-general-changes">General changes</h3>
+
+- Updated Go version to 1.21.3. [#116098][#116098]
+
+<h3 id="v23-2-0-beta-3-sql-language-changes">SQL language changes</h3>
+
+- Added the `sql.ttl.default_select_rate_limit` [cluster setting](../v23.2/cluster-settings.html) and the `ttl_select_rate_limit` [table storage parameter](../v23.2/with-storage-parameter.html#table-parameters) to set the TTL select rate limit. This sets the number of records per table per second per node that can be selected by the TTL job. [#115802][#115802]
+
+<h3 id="v23-2-0-beta-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that could result in an incorrect `too few columns` error for queries that use `ANY <array>` syntax with a subquery. [#115592][#115592]
+- Fixed a bug that could cause `too few columns`/`too many columns` errors for queries that used `IN` or `NOT IN` with a non-trivial right operand, such as a subquery (rather than a constant tuple). [#115592][#115592]
+- Fixed a bug where [`CREATE INDEX`](../v23.2/create-index.html) with expressions could fail on materialized [views](../v23.2/views.html) when the declarative schema changer was used. [#115522][#115522]
+- Fixed a bug that could cause PL/pgSQL routines with `SELECT INTO` syntax to return early. This bug existed only in pre-release versions v23.2.0-beta.1 and v23.2.0-beta.2. [#115676][#115676]
+- Fixed a bug that could cause side effects to happen out of order for PL/pgSQL routines in rare cases. This bug existed only in v23.2 alpha and beta pre-release versions. [#115840][#115840]
```

Comment:
Should we clarify the beta here? Because this is also a beta release, maybe we should add something to the effect that it is beta.2 and earlier?